jovo-platforms/jovo-platform-alexa/src/core/AlexaResponse.ts

```diff
@@ -25,9 +25,18 @@ export interface DynamicEntityType {
   }>;
 }
 
+type YES_NO_MAYBE = 'YES' | 'NO' | 'MAYBE';
+
+interface CanFulfillSlotEntry {
+  canUnderstand: YES_NO_MAYBE,
+  canCullFill: YES_NO_MAYBE,
```

Comment:
Type?

Reply:
Yes. canFulfill is righ.
https://developer.amazon.com/en-US/docs/alexa/custom-skills/implement-canfulfillintentrequest-for-name-free-interaction.html#sample-intent-response